exmaple/anchor-point-test/App.tsx

```diff
@@ -1,21 +1,45 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-import AnchorPointDemo from './AnchorPointDemo';
+import React, { useState } from "react";
```

Comment:
hi, thank you for your contribution. Sorry for the delay reply.  Here i just found i using single quotation marks, while  you using double quotation  mark `""`. A bit hard to find your code changes in this pull request. Could you help convert them to single quotation marks?

Reply:
hi, I removed all formatting modifications.